src/integrations/__tests__/unixIntegrationManager.spec.ts

```diff
@@ -47,287 +43,260 @@ afterEach(async() => {
   }
 });
 
-testUnix('Ensure symlinks and dirs are created properly', async() => {
-  const integrationManager = new UnixIntegrationManager(
-    resourcesDir, integrationDir, dockerCliPluginDir);
+describeUnix('UnixIntegrationManager', () => {
+  let integrationDir: string;
+  let dockerCliPluginDir: string;
+  let integrationManager: UnixIntegrationManager;
 
-  await integrationManager.enforce();
-  expect(fs.promises.readdir(integrationDir)).resolves.not.toThrow();
```

Comment:
missing `await`

Reply:
Huh, didn't know about that. Nice catch!

Reply:
I'm going to remove this call. It was there to check that `UnixIntegrationManager.enforce()` creates `integrationDir`, but that is unnecessary because the checks for each symlink in the for loop below it effectively do that.